javers-core/src/main/java/org/javers/common/reflection/ReflectionUtil.java

```diff
@@ -174,6 +184,21 @@ private static boolean isProtected(Member member){
 
         return Lists.immutableListOf(((ParameterizedType) javaType).getActualTypeArguments());
     }
+    
+    public static Class<?>[] getClasses(Class<?> annotation, String... paths) {
+    	List<String> names = new FastClasspathScanner(paths).scan().getNamesOfClassesWithAnnotation(annotation);
+    	List<Class<?>> classes = new ArrayList<Class<?>>();
+    	for (String c : names) {
+    		Class<?> classType;
+			try {
+				classType = Class.forName(c);
+			} catch (ClassNotFoundException e) {
+				throw new JaversException(JaversExceptionCode.CANT_FIND_CLASS_WITH_TYPENAME, c);
+			}
+    		classes.add(classType);
+    	}
+    	return null;
```

Comment:
this method (clue of your PR) always returns null.
Write tests, check if they fail, and then implement a new feature.